src/component/parser/json/converter/DiagramConverter.ts

```diff
@@ -133,11 +134,12 @@ export default class DiagramConverter extends AbstractConverter<BpmnModel> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private deserializeEdges(edges: any): Edge[] {
     return ensureIsArray(edges).map(edge => {
-      const flow = findSequenceFlow(edge.bpmnElement) || findMessageFlow(edge.bpmnElement);
+      const flow = findSequenceFlow(edge.bpmnElement) || findMessageFlow(edge.bpmnElement) || findAssociationFlow(edge.bpmnElement);
       const waypoints = this.deserializeWaypoints(edge.waypoint);
       const label = this.deserializeLabel(edge.BPMNLabel, edge.id);
       const messageVisibleKind = edge.messageVisibleKind ? edge.messageVisibleKind : MessageVisibleKind.NONE;
-      return new Edge(edge.id, flow, waypoints, label, messageVisibleKind);
+      const associationDirectionKind = edge.associationDirectionKind ? edge.associationDirectionKind : AssociationDirectionKind.NONE;
+      return new Edge(edge.id, flow, waypoints, label, messageVisibleKind, associationDirectionKind);
```

Comment:
Don't forget to update the svg corresponding to the internal model for the doc 😉

Reply:
good point 👍